tests/test_io.py

```diff
@@ -66,10 +66,7 @@
 from pandas.io.sas.sas_xport import XportReader
 from pandas.io.stata import StataReader
 
-from . import (
-    lxml_skip,
-    pytables_skip,
-)
+from . import lxml_skip
```

Comment:
Could this be `from tests import lxml_skip` (if it is in `__init__.py`)?

Reply:
Probably, I just left it as it was.